docs/source/cheat_sheet.rst

```diff
@@ -94,7 +94,14 @@ language represents various common types in Python 2.
        return sum([val for val in my_tuple])
    f((1, 2))
 
-   # TODO: Add Generator example
+   from typing import Generator
+   def f(n):
+       # type: (int) -> Generator
```

Comment:
This should either be `Generator[int, None, None]` or `Iterator[int]` (the latter is simpler, remember to update the import).